packages/@sanity/cli/src/util/noSuchCommandText.js

```diff
@@ -7,6 +7,7 @@ const coreCommands = [
   'build',
   'check',
   'configcheck',
+  'cors',
```

Comment:
Is this intentional? Doesn't seem to be related to cors?

Reply:
I realized it was missing from the array while editing the file - didn't think to open a separate PR for it. 🤷‍♂️

Reply:
Ah, no need for a separate PR I think - but should'nt `graphql` be in that list too?

Reply:
Not yet, we don't want to explicitly hint at it before it's "public"